test/integration/fixtures/01-users.js

```diff
@@ -8,19 +8,22 @@ module.exports = ({ Assignments, Users, Projects, bcrypt }) => {
   const users = [
     new User({ email: 'alice@getodk.org', password: 'alice' }, { actor: new Actor({ type: 'user', displayName: 'Alice' }) }),
     new User({ email: 'bob@getodk.org', password: 'bob' }, { actor: new Actor({ type: 'user', displayName: 'Bob' }) }),
-    new User({ email: 'chelsea@getodk.org', password: 'chelsea' }, { actor: new Actor({ type: 'user', displayName: 'Chelsea' }) })
+    new User({ email: 'chelsea@getodk.org', password: 'chelsea' }, { actor: new Actor({ type: 'user', displayName: 'Chelsea' }) }),
+    new User({ email: 'dave@getodk.org', password: 'dave' }, { actor: new Actor({ type: 'user', displayName: 'Dave' }) }),
   ];
 
-  // hash the passwords, create our three test users, then add grant Alice and Bob their rights.
+  // hash the passwords, create our test users, then grant Alice and Bob their rights.
   const withPasswords = Promise.all(users.map((user) =>
     bcrypt.hash(user.password).then((password) => user.with({ password }))));
 
   return Projects.create(proj)
     .then(() => withPasswords)
     .then((users) => mapSequential(users, Users.create))
-    .then(([ alice, bob, chelsea ]) => Promise.all([
+    .then(([ alice, bob, chelsea, dave ]) => Promise.all([
       Assignments.grantSystem(alice.actor, 'admin', '*'),
-      Projects.getById(1).then((project) => Assignments.grantSystem(bob.actor, 'manager', project.get()))
+      Projects.getById(1).then((project) => Assignments.grantSystem(bob.actor, 'manager', project.get())),
+      Projects.getById(1).then((project) => Assignments.grantSystem(dave.actor, 'formfill', project.get())),
+      Projects.getById(1).then((project) => Assignments.grantSystem(dave.actor, 'viewer', project.get())),
```

Comment:
What do you think about calling `Projects.getById()` just once?

Reply:
E.g.

```
    .then(([ alice, bob, chelsea, dave ]) => Promise.all([
      Assignments.grantSystem(alice.actor, 'admin', '*'),
      Projects.getById(1).then((project) => Promise.all([
        Assignments.grantSystem(bob.actor, 'manager', project.get()),
        Assignments.grantSystem(dave.actor, 'formfill', project.get()),
        Assignments.grantSystem(dave.actor, 'viewer', project.get()),
      ])),
    ]));
```

?  Maybe more efficient?  Is it more readable?